src/combinatorics/simple_graph/clique.lean

```diff
@@ -140,6 +195,14 @@ end
 lemma clique_free.anti (h : G ≤ H) : H.clique_free n → G.clique_free n :=
 forall_imp $ λ s, mt $ is_n_clique.mono h
 
+/-- See `simple_graph.clique_free_chromatic_number_succ` for a tighter bound. -/
+lemma clique_free_card_succ [fintype α] : G.clique_free (fintype.card α + 1) :=
```

Comment:
I'm not sure whether this or the next is more useful. I'm tempted to say the next, because you don't have to deal with the annoying rewrite `a + 1 ≤ b ↔ a < b`.
```suggestion
lemma clique_free_of_card_lt [fintype α] (h : card α < n) : G.clique_free n :=
```

Reply:
Good idea, I switched it over.